spec/system/admin/admin_creates_new_page.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe "Admin creates new page", type: :system do
```

Comment:
I'm a bit confused, is this spec really related to the changes in this PR?

Reply:
these changes updated due to conflict. I think this file is deleted in the master branch. I will check and update this.